client/package.json

```diff
@@ -4,8 +4,8 @@
   "author": "Nadav Ivgi",
   "license": "MIT",
   "scripts": {
-    "postinstall": "../node_modules/.bin/patch-package",
-    "dist": "../node_modules/.bin/babel -d dist src"
+    "postinstall": "cd ../node_modules/.bin &&  patch-package",
```

Comment:
```suggestion
    "postinstall": "cd ../node_modules/.bin && ./patch-package",
```

Reply:
Actually sir, `./` was causing problem in windows.
With the errors like :`.` `is not recognized as a command` .
and not adding the `./`was working fine in linux.